frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -2900,6 +2901,21 @@ static bool resolveFnCallSpecial(Context* context,
     }
   }
 
+  if (ci.isOpCall() && ci.name() == USTR(":")) {
+    auto src = ci.actual(0).type();
+    auto dst = ci.actual(1).type();
+
+    if (src.isType() && dst.hasTypePtr() && dst.type()->isStringType()) {
+      std::ostringstream oss;
+      src.type()->stringify(oss, chpl::StringifyKind::CHPL_SYNTAX);
```

Comment:
No changes are required but I wanted to note that CHPL_SYNTAX does not always produce valid chapel syntax today (and sometimes is very odd). I'll open an issue and assign myself since this is relevant to other areas of dyno too.